presto-tests/src/test/java/com/facebook/presto/tests/TestLocalQueries.java

```diff
@@ -88,9 +88,9 @@ public void testShowColumnStats()
         MaterializedResult expectedStatistics =
                 resultBuilder(getSession(), VARCHAR, DOUBLE, DOUBLE, DOUBLE, DOUBLE, VARCHAR, VARCHAR)
                         .row("nationkey", null, 25.0, 0.0, null, "0", "24")
-                        .row("name", 177.0, 25.0, 0.0, null, "ALGERIA", "VIETNAM")
+                        .row("name", 177.0, 25.0, 0.0, null, null, null)
```

Comment:
Could you update the commit message to mention that SHOW STATS no longer prints min/max for varchar columns.